src/services/user.ts

```diff
@@ -256,13 +256,10 @@ const setHomePhonePrivacy = (makePrivate: boolean) =>
 const setImagePrivacy = (makePrivate: boolean) =>
   http.put('profiles/image_privacy/' + (makePrivate ? 'N' : 'Y')); // 'Y' = show image, 'N' = don't show image
 
-const getBirthdate = async (): Promise<DateTime> =>
-  DateTime.fromISO(await http.get('profiles/birthdate'));
+const getBirthdate = async (): Promise<Date> =>
```

Comment:
Since this function does not use the `await` keyword, it does not need to be `async`.
```suggestion
const getBirthdate = (): Promise<Date> =>
```

Because `async` functions always return a Promise, even if the function never actually `await`s anything, this is adding an extra Promise object for no benefit.